Shared/Samples/Analyze network with subnetwork trace/AnalyzeNetworkWithSubnetworkTraceView.swift

```diff
@@ -0,0 +1,267 @@
+                }
+            }
+            .overlay(alignment: .center) {
+                if let statusText = model.statusText {
+                    ZStack {
+                        Color.clear.background(.ultraThinMaterial)
+                        VStack {
+                            Text(statusText)
+                            ProgressView()
+                                .progressViewStyle(.circular)
+                        }
+                        .padding()
+                        .background(.ultraThickMaterial)
+                        .cornerRadius(10)
+                        .shadow(radius: 50)
+                    }
+                }
+            }
+    }
+    
+    @ViewBuilder var settingsView: some View {
+        Form {
+            Section("Trace Options") {
+                Toggle("Includes barriers", isOn: $includesBarriers)
+                Toggle("Includes containers", isOn: $includesContainers)
+            }
+            Section {
+                ForEach(model.conditions, id: \.self) { condition in
+                    Text(condition)
+                }
+                .onDelete { indexSet in
+                    model.deleteConditionalExpression(atOffsets: indexSet)
+                }
+            } header: {
+                Text("List of conditions")
+            } footer: {
+                Text(model.expressionString)
+            }
+        }
+    }
+    
+    @ViewBuilder var conditionMenu: some View {
+        List {
+            NavigationLink {
+                attributesView
+            } label: {
+                Text("Attributes")
+                Spacer()
+                Text(model.selectedAttribute?.name ?? "")
+                    .frame(maxWidth: .infinity, alignment: .trailing)
+            }
+            NavigationLink {
+                operatorsView
+            } label: {
+                Text("Comparison")
+                Spacer()
+                Text(model.selectedComparison?.title ?? "")
+                    .frame(maxWidth: .infinity, alignment: .trailing)
+            }
+            if model.selectedAttribute?.domain as? CodedValueDomain != nil {
+                HStack {
+                    NavigationLink {
+                        valuesView
+                    } label: {
+                        Text("Value")
+                        Spacer()
+                        if let value = model.selectedValue as? CodedValue {
+                            Text(value.name)
+                                .frame(maxWidth: .infinity, alignment: .trailing)
+                        }
+                    }
+                }
+                .disabled(model.selectedAttribute == nil && model.selectedComparison == nil)
+            } else {
+                HStack {
+                    Text("Value")
+                    Spacer()
+                    if let value = inputValue {
+                        Text(value.formatted())
+                    }
+                }
```

Comment:
Could a text field be used instead of an alert? That's a much more idiomatic way of getting the input value.

```swift
HStack {
    Text("Value")
    TextField("Comparison value", value: $inputValue, format: .number, prompt: Text("Value"))
        .multilineTextAlignment(.trailing)
        .lineLimit(1)
}
```